docs/user-guide/api-mediation/api-mediation-internal-configuration.md

```diff
@@ -101,7 +101,19 @@ This section describes runtime configuration properties.
 * **server.maxTotalConnections and server.maxConnectionsPerRoute**
 
     These two properties are used to set the number of concurrent connections. Any connection requests that are made that would put the number of connections over either of these limits are queued until an existing connection completes. The API Gateway is built on top of Apache HTTP components that require these two connection limits for concurrent requests. For more information, see [Apache documentation](http://hc.apache.org/httpcomponents-client-ga/tutorial/html/connmgmt.html#d5e393).
+## Environment variables
 
+Users can configure some properties through environment variables in the instance.env file. Gateway service will inject them to fields. Environment variables don't have the highest priority in this context. That means, environment variable will be overridden with command line argument provided at the start of the application. Such arguments are in the start script and are prefixed with `-D`.
```

Comment:
It's a bit unclear to me. 

I would probably focus on something like. The internal configuration isn't exposed via Zowe wide UPPER_CASE named variables with default provided in the instance.env. It's possible to change it via the environment variables nevertheless. The way to do it is this.